app/Events/PlayerJoinedGameEvent.php

```diff
@@ -15,8 +15,11 @@ class PlayerJoinedGameEvent extends GameEvent
 
     public function __construct($gameId, User $user)
     {
-        $this->user = $user;
-        $this->gameId = $gameId;
+        parent::__construct($gameId);
```

Comment:
Dit zorgde ervoro dat we in de app niet wisten wat we met deze speler aan moesten